core/kazoo_data/src/kzs_plan.erl

```diff
@@ -115,13 +115,32 @@ get_dataplan(DBName, DocType, DocOwner) ->
         Else -> system_dataplan(DBName, Else)
     end.
 
+system_connection_map(Connections) ->
+    ServerMap = #{tag => 'local'
+                 ,server => maps:get('local', maps:from_list(Connections))
+                 ,others => [{Tag, #{tag => Tag, server => Server}} || {Tag, Server} <- Connections, Tag =/= 'local']
+                 },
+    cache_dataconnection(?KAZOO_DATA_PLAN_CACHE, {'connection', <<"system">>, 'undefined'}, ServerMap),
```

Comment:
This call also viiolates the spec of the `cache_dataconnection/3` function.

Should probably put `{'connection', <<"system">>, 'undefined'}` into a macro like `-define(SYSTEM_DATACONNECTION_KEY, {'connection'...}).` - seems like it improves readability (at least to me).